install-dependencies/README.md

```diff
@@ -0,0 +1,59 @@
+# GitHub Action Install Dependencies
+
+<!-- prettier-ignore-start -->
+<!-- action-docs-description source="action.yaml" -->
+## Description
+
+GitHub Action that installs dependencies in a Node based repository
```

Comment:
Let's add a note on the README so we know that providing `s3` params requires authentication to be done beforehand